baseapp/baseapp.go

```diff
@@ -31,6 +31,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/types/mempool"
+	simcli "github.com/cosmos/cosmos-sdk/x/simulation/client/cli"
```

Comment:
This feels not right to import a `x/` path in baseapp.

Reply:
Yes, there is some truth to what you said. But as I mentioned above: "**Add the Sigverify flag in the simulation test whether to check the transaction signature.**", in order to achieve this requirement, I followed the previous implementation and added a **FlagSigverify** global variable（x/simulation/client/cli/flags.go）, and during in the simulation the test, I need to verify the user's signature based on this global variable. In order to achieve this requirement, I have not thought of a better implementation for the time being. Can you give me any more suggestions or tips?

Reply:
> This feels not right to import a `x/` path in baseapp.

@julienrbrt A private variable sigverifyTx has been added to the `type BaseApp struct` to solve this problem. Please help me review the code again.